samples/java-customer-registry-views-quickstart/src/main/proto/customer/view/customer_view.proto

```diff
@@ -12,61 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// tag::declarations[]
-syntax = "proto3"; // <1>
+syntax = "proto3";
 
-package customer.view; // <2>
+package customer.view;
```

Comment:
wasn't really used in docs or why is the callouts not needed anymore?

Reply:
No, apparently not. Docs are passing.

Reply:
See comment above on the other proto files